disnake/member.py

```diff
@@ -427,6 +431,7 @@ def _update(self, data: GuildMemberUpdateEvent) -> None:
         self._avatar = data.get("avatar")
         timeout_datetime = utils.parse_time(data.get("communication_disabled_until"))
         self._communication_disabled_until = timeout_datetime
+        self._flags = data["flags"]
```

Comment:
`_update_from_message` is missing this change

Reply:
a68ddb62a99a4df3860fd97cb775a95f84714b65